lib/components-schema-v1_8_x.ts

```diff
@@ -665,6 +665,27 @@ export const componentsDefinitionSchema_v1_8_x: JSONSchema7 = {
                 type: 'string',
             },
         },
+
```

Comment:
Added this part to the 1.8 next schema. Is this fine or should a new 1.9 schema already be added?

Reply:
Yes, that's the purpose of the "-next" schema: https://github.com/WoodWing/studio-component-set-tools#releasing-a-new-schema-version

In case the readme is not clear enough, maybe improve it?

Reply:
I think the readme is clear. But I didn't know in what version the conditional properties were released. As those are in 1.7.0 it turns out this feature is actually 1.8.0.